doc/src_usermanual/techrep_main.tex

```diff
@@ -926,10 +925,10 @@ \subsection{Linear algebra computational kernels}
 
 \noindent \textbf{mem\_space}: specifies the primary memory space in which the RAJA-based optimization and linear algebra objects will be created
 \begin{itemize}
-\item ``default'' (default): the default memory space as per RAJA specification 
-\item ``host'': cpu memory space
-\item ``device'': device memory space
-\item ``um'': unified memory space
+\item ``default'' (default): allocations are done by \Hi in the cpu's memory space
+\item ``host'':  allocations via UMPIRE in UMPIRE's ``HOST`` memory space, typically CPU memory
+\item ``device'': allocations via UMPIRE in UMPIRE's ``DEVICE`` device memory space
+\item ``um'': allocations via UMPIRE's unified memory model, know as ``UM`` 

```

Comment:
Fix Umpire name, add periods at the end of each bullet, and fix LaTeX quotation marks:
```suggestion
\item ``default'' (default): allocations are done by \Hi in the cpu's memory space.
\item ``host'':  allocations in Umpire's ``HOST'' memory space, typically CPU memory.
\item ``device'': allocations in Umpire's ``DEVICE'' device memory space.
\item ``um'': allocations in Umpire's unified memory space ``UM''.

```

Reply:
done